### PR TITLE
Add a flag to enable NuGet.org README.md display

### PR DESCRIPTION
**Pull request by pakrym**
Contributes to https://github.com/Azure/azure-sdk-for-net/issues/21160

### CODE REVIEW

**Comment by pakrym** [comment]
Requires new SDK will wait a bit.

File: global.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "5.0.200",
+    "version": "5.0.300-preview.21180.15",
```
Comment (weshaggard) [inline global.json:6]: Yes lets please wait until there is a release SDK that supports this scenario.

**Comment by pakrym** [comment]
Couple problems:

1. HTML-style comments are displayed.
2. Common bar display.
image
3. Link labels are not removed 
image

**Comment by pakrym** [comment]
I think we can pre-process the markdown ourselves. But I'll file an issue on Nuget first and see if they are open to making changes in their generation.

**Comment by pakrym** [comment]
Filed https://github.com/NuGet/NuGetGallery/issues/8627 but also added a pre-process script.

**Review by weshaggard — APPROVED** [review]
Looks reasonable once folks are OK with moving to the new toolset.

File: global.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "5.0.200",
+    "version": "5.0.300",
```
[the remark this answers is not in the document]
Comment (weshaggard) [reply global.json:6]: We should also update the Contributing.md https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#prerequisites

**Pull request merged**